### ISSUE TITLE
false positive indentation alert for sass map used as mixin argument

### ISSUE THREAD

**Issue by sormy** [sasstools/sass-lint#927]
scss code:

```
@include btn-variations((
  primary: (
    background-color: $orange
  ),
  secondary: (
    background-color: $gray-blue
  )
));
```

sass-lint@1.9.1 is used within Visual Studio Code plugin.

Will show false positive "expected indentation of 4 spaces but found 2" for "primary: (" line. It is expected that line "primary: (" should have 2 spaces correct indentation.

.sass-lint.yml

```yml
rules:
  indentation:
    - 2
    - { size: 2 }
  property-sort-order: 0
```

**Comment by DanPurdy** [comment]
I'll look into it @sormy there's still a few edge cases left over to sort with indentation. 

Just out of interest does it still report this if you use this in your config file

```yaml
rules: 
  indentation: 
    - 2
    - 
      size: 2
  property-sort-order: 0
```

Thanks

**Comment by hotforfeature** [comment]
@DanPurdy that is how my config looks and I still get the error
